src/config/reader.rs

```diff
@@ -116,20 +116,32 @@ impl ConfigReader {
                     }
                 }
                 LinkType::Protobuf => {
-                    let mut descriptors = self
-                        .resolve_descriptors(self.read_proto(&source.path).await?)
-                        .await?;
+                    let parent_descriptor = self.read_proto(&source.path).await?;
+                    let id = parent_descriptor
+                        .package // we should ideally take the package name as default id
```

Comment:
there was some attempt to specify fully qualified method name without linking to file explicitly and just search for the required definition across all files. 

Here https://github.com/tailcallhq/tailcall/pull/1172/files#diff-292fd7f2de6455aa7a8afbf0b7a1d0f4a7538f4f1c92ba3be3e29b914d749867
wdyt about this approach?

Reply:
I agree